public/docs/ts/_cache/tutorial/toh-pt6.jade

```diff
@@ -183,173 +209,141 @@ block get-heroes-details
 
 - var rejected_promise = _docsFor == 'dart' ? 'propagated exception' : 'rejected promise';
 :marked
-  In this demo service we log the error to the console; we should do better in real life.
+  In this demo service we log the error to the console; we would do better in real life.
 
   We've also decided to return a user friendly form of the error to
   the caller in a !{rejected_promise} so that the caller can display a proper error message to the user.
 
-  ### !{_Promise}s are !{_Promise}s
+  ### Unchanged `getHeroes` API
+
   Although we made significant *internal* changes to `getHeroes()`, the public signature did not change.
   We still return a !{_Promise}. We won't have to update any of the components that call `getHeroes()`.
 
-.l-main-section
-:marked
-  ## Add, Edit, Delete
+  ### Add, edit, delete heroes
```

Comment:
This heading used to be directly tied to the order of the sections to follow (post, put, delete, and then a save section that combined the first two). Now I don't even see "post(" in the Dart page unless I select lib/hero_service.dart at the bottom of the page.

I think this rewrite needs some more connecting material, plus a careful review to make sure it includes everything it needs.

(Which reminds me... I noticed that some of the earlier samples got longer, so we need to also make sure that this page doesn't include anything unnecessary. `/*...*/` can be a good thing, reducing complexity and scariness.)